src/redux/baseApi.tsx

```diff
@@ -0,0 +1,13 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+
+export const REDUCER_PATH = "api";
+export const SPOTIFY_REDUCER_PATH = "spotify_api";
+
+export const tagTypes = [] as const; // TODO: will add tags here
+
+export const baseApi = createApi({
+  reducerPath: REDUCER_PATH,
+  baseQuery: fetchBaseQuery({ baseUrl: "http://localhost:8000/api/v1/" }), //TODO: will use this endpoint from env file when multiple env exists
```

Comment:
As some of us might not use localhost domain, Let's use the endpoint from env from the start, It Should not be a big lift as env is already setup and being used.